src/movies.js

```diff
@@ -1,25 +1,131 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+    const directors = moviesArray.map(el => el.director);
+    return directors;                      
+}
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies(moviesArray) {}
+function howManyMovies(moviesArray) {
+    const spielbergMovies = moviesArray.filter(el => el.director === "Steven Spielberg" && el.genre.includes("Drama"));
+    return spielbergMovies.length;
```

Comment:
Igual!! de 10!, refactor pero leve!

```suggestion
    return moviesArray.filter(el => el.director === "Steven Spielberg" && el.genre.includes("Drama")).length;
```